src/chromyRunner.js

```diff
@@ -105,7 +106,8 @@ const run = async (chromy, options, test) => {
       case actions.freezeImage:
         try {
           prefixedLogger.log(`Freezing image ${action.value} in browser`);
-          const result = await chromy.evaluate(() => freezeImage(action.value));
+          const strFunc = functionToString(freezeImage, { '{{selector}}': action.value });
+          const result = await chromy.evaluate(`return (${strFunc})()`);
```

Comment:
The `return (` and `)()` should be inserted by `functionToString`, you will always want to put those there, right?